e2e/go.mod

```diff
@@ -193,6 +193,17 @@ require (
 replace (
 	github.com/openclarity/vmclarity => ../
 	github.com/openclarity/vmclarity/api => ../api
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc => go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.40.0
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.40.0
+
+	// These are replaced in order to be compatible with docker compose v2.22.0
```

Comment:
I would group replaces by component instead of smashing everything together. Could you please move the docker-compose related replaces back to a separate group which we already have.
Also it would be really helpful to add a bit of context in comments why we need these replaces and which dependencies requiring them. It will make the job of the next person looking into this issue way easier when they need to figure out if the replaces are still valid or need to be updated, etc.